src/gui/properties/propertieswidget.cpp

```diff
@@ -785,9 +785,11 @@ void PropertiesWidget::askWebSeed()
         return;
     }
     if (m_torrent)
+    {
         m_torrent->addUrlSeeds({urlSeed});
-    // Refresh the seeds list
-    loadUrlSeeds();
+        // Refresh the seeds list
+        loadUrlSeeds();
```

Comment:
wait... if you are trying to guard `m_torrent` from being `nullptr` then you should done it within `loadUrlSeeds()` instead, not here.

Reply:
>you should done it within loadUrlSeeds() instead, not here.

It would look more reliable.

Reply:
Is this a requested change or not? All three of us have approved this PR but it isn't merged yet due to this ambiguity.
Personally I don't have problem merging it as is.

Reply:
>Is this a requested change or not? 

It is, at least it calls for reevaluation of the proposed change.

Reply:
As I can see there are several methods that use m_torrent directly but have no check for `nullptr`. All of them are flaws (at least, potential ones).